example/ios/Podfile.lock

```diff
@@ -437,10 +439,10 @@ PODS:
     - React-RCTText
     - ReactCommon/turbomodule/core
     - Yoga
-  - RNScreens (3.19.0):
+  - RNScreens (3.20.0):
     - React-Core
     - React-RCTImage
-  - RNSVG (13.8.0):
+  - RNSVG (13.9.0):
```

Comment:
Just to make sure - are these other pod updates intentional?  Not opposed to them, but from a purity perspective it'd be sweet to see this file be nothing but `react-native-notifications` related.

Reply:
No, they are not intentional. I reverted the cocoapods version to `1.11.3`, and did a full re-install. We're not pinning the corresponding libraries to specific versions, which is I believe these pods got bumped up.